openpolicedata/data_loaders.py

```diff
@@ -104,6 +104,61 @@ def load_csv(url, date_field=None, year_filter=None, agency_field=None, agency=N
         agency_field=agency_field, agency=agency)
 
     return table
+def load_excel(url, date_field=None, year_filter=None, agency_field=None, agency=None, limit=None, pbar=True):
+    '''Download XLS file to pandas DataFrame
+    
+    Parameters
+    ----------
+    url : str
+        Download URL for Excel
+    date_field : str
+        (Optional) Name of the column that contains the date
+    year_filter : int, list
+        (Optional) Either the year or the year range [first_year, last_year] for the data that is being requested. None value returns data for all years.
+    agency_field : str
+        (Optional) Name of the column that contains the agency name (i.e. name of the police departments)
+    agency : str
+        (Optional) Name of the agency to filter for. None value returns data for all agencies.
+    limit : int
+        (Optional) Only returns the first limit rows of the CSV
+    pbar : bool
+        (Optional) This progress bar argument is currently ignored. 
+        
+    Returns
+    -------
+    pandas DataFrame
+        DataFrame containing table imported from Excel spreadsheet
+    '''
+    if ".zip" in url:
```

Comment:
Has this code been tested? If not, I recommend deleting support for .zip files. I'm not aware of any URLs with a zipped Excel to test this on.

Reply:
made the change